packages/vats/src/vat-board.js

```diff
@@ -1,7 +1,23 @@
-import { Far } from '@endo/far';
-import { makeBoard } from './lib-board.js';
+// @ts-check
+import { M, prepareExo } from '@agoric/vat-data';
+import { prepareBoardClass } from './lib-board.js';
 
-export function buildRootObject() {
-  const board = makeBoard();
-  return Far('board', { getBoard: () => board });
+const RootI = M.interface('VatBoard', {
+  getBoard: M.call().returns(M.remotable()),
+});
+
+/**
+ * @param {unknown} _vatPowers
+ * @param {unknown} _vatParameters
+ * @param {import('@agoric/vat-data').Baggage} baggage
+ */
+export function buildRootObject(_vatPowers, _vatParameters, baggage) {
+  const makeBoardKit = prepareBoardClass(baggage);
```

Comment:
I think our scheme has been `makeFoo` is returned by a `prepareFoo`.
```suggestion
  const makeBoardKit = prepareBoardKit(baggage);
```

Reply:
I must admit that after poring over the naming scheme and trying a few different approaches, I got lazy and assumed you would tell me the right name :)